mgmt/imgmgr/src/imgmgr.c

```diff
@@ -854,29 +854,38 @@ imgr_upload(struct mgmt_cbuf *cb)
 }
 
 void
-imgr_dfu_stopped(void)
+imgmgr_dfu_stopped(void)
 {
-    if (imgmgr_dfu_callbacks_fn->dfu_stopped_cb != NULL)
+    if (imgmgr_dfu_callbacks_fn && imgmgr_dfu_callbacks_fn->dfu_stopped_cb != NULL)
     {
         imgmgr_dfu_callbacks_fn->dfu_stopped_cb();
     }
 }
 
 void
-imgr_dfu_started(void)
+imgmgr_dfu_started(void)
 {
-    if (imgmgr_dfu_callbacks_fn->dfu_started_cb != NULL)
+    if (imgmgr_dfu_callbacks_fn && imgmgr_dfu_callbacks_fn->dfu_started_cb != NULL)
     {
```

Comment:
I know this is was not changed in this commit, but since "if" line is changed maybe adjusting opening { placement could also be corrected. Same for other places that slipped through previous review.

Reply:
Yes, this has been fixed now